test/JDBC/expected/BABEL-2349.out

```diff
@@ -39,6 +39,45 @@ int
 use master;
 go
 
+-- sysindexes should also exist in dbo schema
+-- If there are white spaces between schema name and catalog name then those need to be ignored
+-- case insensitive check
+SELECT COUNT(*) FROM sys.    SySInDeXes where id=OBJECT_ID('t1');
+go
+~~START~~
+int
+2
```

Comment:
Are we hitting the same execution path when we use the same statements using prepared statements? Can you please add test cases for that.